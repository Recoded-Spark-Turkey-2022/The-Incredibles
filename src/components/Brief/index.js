import Button from '../../components/UI/Button';
import { useAuthState } from 'react-firebase-hooks/auth';
import { auth } from '../../firebase/firebase';
import HomeImage from '../../assets/pics/homepage/hero.svg';
import { useTranslation } from 'react-i18next';
import Container from '../UI/Container';

function Brief() {
  const [t] = useTranslation();
  const [user] = useAuthState(auth);

  return (
    <Container>
      <div
        name="main"
        className="flex flex-col-reverse md:flex-row md:justify-between md:h-auto "
      >
        <div name="wirtting" className="md:w-1/2 md:text-left text-center">
          <h1 className="font-bold md:text-7xl text-cyan-600 text-5xl my-5">
            {t('home.home.h1')}
          </h1>
          <h2 className="md:text-5xl text-3xl  my-5 text-rose-400 font-medium ">
            {t('home.home.h2')}
          </h2>
          <div className="">
            <p className="sm:text-xl text-base">{t('home.home.p')}</p>
          </div>

          <div className={user ? 'hidden' : 'my-5'}>
            <Button name={t('nav.signup')} path="/signup" />
          </div>
        </div>
        <div name="image" className="m-auto md:w-1/2">
          <img src={HomeImage} alt="logo" className="" />
        </div>
      </div>
    </Container>
  );
}
export default Brief;
