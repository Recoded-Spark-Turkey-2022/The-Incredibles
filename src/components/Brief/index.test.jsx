import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Brief from '.';

describe('Brief', () =>
  it('renders correctly', () => {
    const domTree = renderer
      .create(
        <Router>
          <Brief />
        </Router>
      )
      .toJSON();
    expect(domTree).toMatchSnapshot();
  }));
