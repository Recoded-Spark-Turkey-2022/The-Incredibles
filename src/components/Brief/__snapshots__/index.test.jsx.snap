// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brief renders correctly 1`] = `
<div
  className="container max-w-screen-xl m-auto p-10 max-md:px-5"
>
  <div
    className="flex flex-col-reverse md:flex-row md:justify-between md:h-auto "
    name="main"
  >
    <div
      className="md:w-1/2 md:text-left text-center"
      name="wirtting"
    >
      <h1
        className="font-bold md:text-7xl text-cyan-600 text-5xl my-5"
      >
        home.home.h1
      </h1>
      <h2
        className="md:text-5xl text-3xl  my-5 text-rose-400 font-medium "
      >
        home.home.h2
      </h2>
      <div
        className=""
      >
        <p
          className="sm:text-xl text-base"
        >
          home.home.p
        </p>
      </div>
      <div
        className="my-5"
      >
        <button
          className="px-10 py-2.5 bg-cyan-600 text-white font-medium text-l leading-tight
 rounded-full shadow-md
 ease-in duration-300 hover:bg-purple-700 hover:shadow-lg hover:scale-110"
          onClick={[Function]}
          type="button"
        >
          nav.signup
        </button>
      </div>
    </div>
    <div
      className="m-auto md:w-1/2"
      name="image"
    >
      <img
        alt="logo"
        className=""
        src="hero.svg"
      />
    </div>
  </div>
</div>
`;
